geometry/rotation_body.hpp

```diff
@@ -59,26 +61,30 @@ FORCE_INLINE Quaternion ToQuaternion(R3x3Matrix const& matrix) {
   return Quaternion(real_part, imaginary_part);
 }
 
+// Returns a rotation of |angle| around |axis|.  |axis| must be normalized.
+Quaternion AngleAxis(Angle const& angle, R3Element<double> const& axis) {
+  quantities::Angle const half_angle = 0.5 * angle;
+  return Quaternion(Cos(half_angle), Sin(half_angle) * axis);
+}
+
+// Returns a rotation of |angle| around the |axis_index|th axis.
+Quaternion AngleAxis(Angle const& angle, int const axis_index) {
```

Comment:
Even then I'd want to factor `AngleAxis(α, BasisVector(static_cast<int>(axes) >> (2 * 2)) & 0b11))`